ETC.tsv

```diff
@@ -5267,9 +5267,9 @@ ETC_20150317_005266	Over here, teacher!
 ETC_20150317_005267	I was just about to try remaking the inscription rubbing..
 ETC_20150317_005268	We finally did it!
 ETC_20150317_005269	Chapparition appeared while rubbing the inscription!{nl}Please defeat Chapparition!
-ETC_20150317_005270	An uncontrollable Necroventer was summoned.{nl}The experiment has failed. Please defeat Necroventer!
+ETC_20150317_005270	An uncontrollable Necroventer was summoned.{nl}The experiment has failed. Please defeat the Necroventer!
 ETC_20150317_005271	Uncontrollable Necroventer
-ETC_20150317_005272	Please defeat Unicorn together with Soldier's Soul!
+ETC_20150317_005272	Please defeat the Unicorn together with the Soldier's Soul!
```

Comment:
I believe this should remain unedited since the previous line is fine.

Unicorn is the name of the boss monster. By adding 'the', it'll sound odd. Soldier's Soul is probably an NPC who will aid you in the battle of Unicorn.